admin/commands/execution/stop_at_height.go

```diff
@@ -0,0 +1,81 @@
+package execution
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/onflow/flow-go/admin"
+	"github.com/onflow/flow-go/admin/commands"
+	"github.com/onflow/flow-go/engine/execution/ingestion"
+)
+
+var _ commands.AdminCommand = (*StopAtHeightCommand)(nil)
+
+// StopAtHeightCommand will send a signal to engine to stop/crash EN
+// at given height
+type StopAtHeightCommand struct {
+	stopAtHeight *ingestion.StopAtHeight
+}
+
+func NewStopAtHeightCommand(sah *ingestion.StopAtHeight) *StopAtHeightCommand {
+	return &StopAtHeightCommand{
+		stopAtHeight: sah,
+	}
+}
+
+type StopAtHeightReq struct {
+	height uint64
+	crash  bool
+}
+
+func (s *StopAtHeightCommand) Handler(ctx context.Context, req *admin.CommandRequest) (interface{}, error) {
```

Comment:
once you have the error returns of https://github.com/onflow/flow-go/blob/3c6f7bdfd479cd12b209e48011196ddccdde8704/engine/execution/ingestion/stop_at_height.go#L54 documented, you can just carry the doc over here.